invenio_app_ils/ui/src/invenio_app_ils/common/api/locations/internalLocation.js

```diff
@@ -23,5 +28,6 @@ const list = async (query = '') => {
 export const internalLocation = {
   list: list,
   get: get,
+  del: del,
```

Comment:
Can we just rename the exposed name as `delete`? This should be fine no?

Reply:
It will be fine!